### ISSUE TITLE
Spelling mistake in defer.xhtml howto

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6416]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#6416|
|Type|defect|
|Created|2013-04-07 12:40:15Z|

`doc/core/howto/defer.xhtml` has a spelling mistake: "asychronous" rather than "asynchronous".

Attachments:

* [ticket_6416.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/461/4613f9d0a58a3c9ce487b92d94ab3d58768ae608/2f4d9dac0349fd619552cb807cdb79faffef3714.patch) (1305 bytes) - added by kizulha on 2013-04-08 09:20:48Z - Fixes the spelling error

<details><summary>Searchable metadata</summary>

```
trac-id__6416 6416
type__defect defect
reporter__itamar itamar
priority__lowest lowest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1365338415000000 1365338415000000
changetime__1365771650000000 1365771650000000
version__None None
owner__thijs thijs

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks, I'll apply it.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [38046]) Apply ticket_6416.patch: Correct a typo in the Deferred reference document.

Author: kizulha
Reviewer: thijs
Fixes: [#6416](https://github.com/twisted/twisted/issues/6416)
